lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -445,7 +460,7 @@ def generate(cls, _xmodule_instance_args, _entry_id, course_id, _task_input, act
         graded_scorable_blocks = cls._graded_scorable_blocks_to_header(course_id)
 
         # Just generate the static fields for now.
-        rows = [list(header_row.values()) + ['Enrollment Status', 'Grade'] + _flatten(graded_scorable_blocks.values())]
+        rows = [list(header_row.values()) + ['Nombre', 'Fecha Matricula', 'Enrollment Status', 'Nota Final'] + _flatten(graded_scorable_blocks.values())]
```

Comment:
Ideally we will use their i18n module for these strings

Reply:
There fields were added, in addition, a csv is generated and no gettext is loaded

Reply:
Is it possible to introduced gettext so that we use English instead?

Reply:
I will try it